src/BaseUpgradeableBeacon.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v5.0.0) (proxy/beacon/UpgradeableBeacon.sol)
```

Comment:
```suggestion
// Inspired by OpenZeppelin Contracts (last updated v5.0.0) (proxy/beacon/UpgradeableBeacon.sol)
```

Reply:
Updated in this [commit](https://github.com/DistributedCollective/sovryn-lb-dex/pull/3/commits/e7e76f053b2778191a3461452b40d2f0dfe647e7)